addons/core/translations/resources/en-us.yaml

```diff
@@ -912,6 +912,12 @@ worker:
     ip_address: IP Address
     release_version: Release Version
     tag_count: '{tagCount, plural, =1 {# tag} other {# tags}}'
+    key: Key
```

Comment:
I am realizing now I have some duplication of translations between this PR and my other [PR](https://github.com/hashicorp/boundary-ui/pull/2355). I will correct this once one of them is merged.